docs/static/dead-letter-queues.asciidoc

```diff
@@ -0,0 +1,155 @@
+[[dead-letter-queues]]
+=== Dead Letter Queues
+
+//REVIEWERS: I had to install logstash-input-dead_letter_queue. Is it not bundled with the alpha2 release?
+
+NOTE: The dead letter queue feature is currently supported for the Elasticsearch
+output only. Support for additional outputs will be available in future releases
+of the Logstash plugins. 
+
+//REVIEWERS: I feel like we have to say something here ^^ but I'm not sure if this is enough info. How will users be able to tell if a specific output supports DLQs? Do we have a plan for when/how we will add DLQ support to plugins that we support?
+
```

Comment:
Does it make sense to refer the user to the documentation for the plugins? Assume that a plugin isn't supported unless the plugin docs state explicit dlq support. @acchen97, @suyograo may have a rollout plan for plugins.